### DISCUSSION TITLE
Sub-account or account envelope

### DISCUSSION THREAD

**Discussion by Markobros** [moneymanagerex/moneymanagerex#6359] [Ideas]
Hello.
Have you thought about a bank sub-accounts or envelopes under bank accounts - it's only different description for the same.
The behavior should be the same as categories and sub-categories.
Let's have bank account to which one or more sub-accounts can be added.
When adding a new transaction you can choose whether it belongs to the main account or to one of the sub-account.
The account balance of the main bank account then of course consists of the transactions of the main account and all the transactions of the sub-accounts under the main account.

**Comment by whalley** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/3904

**Reply by Macaulay1234** [reply]
Add Bank account
